deluge/tests/test_client.py

```diff
@@ -75,6 +75,9 @@ def __on_disconnect(self):
 
 class ClientTestCase(BaseTestCase, DaemonBase):
 
+    if windows_check:
+        skip = 'windows cant start_core not enough arguments for format string'
```

Comment:
I just realised we should use the unittest `skipIf`: https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures